### ISSUE TITLE
?Feature: Highlight sponsors lectures/workshops on the programme index page

### ISSUE THREAD

**Issue by IGORnvk** [HZ-HBO-ICT/it-conference#593]
Maybe something similar that we do in the speakers page? Like highlight borders with a color of the sponsorship?

**Comment by TimKardol** [comment]
Agreed. I think the way the program is structured is also up for a redesign. I will make a separate issue for that.

**Comment by v-stamenova** [comment]
I think closed with #260?
